docs/ios/6x/getting-started/dsym-upload.md

```diff
@@ -86,7 +86,9 @@ The upload utility is available in the support utility download. You can upload
 
 ### CI/CD Integration
 
-This process can be scripted into your CI backend. Include the upload utility with your project's repository and call it from within your CI scripting system.
```

Comment:
I'm trying to steer people away from committing the actual binary into their project, it'd be better if they re-downloaded the latest .zip every time, so we don't end up getting stuck with people running ancient versions.